check_for_new_releases.py

```diff
@@ -13,9 +13,12 @@ def get_subdirectories(d: str) -> list:
 
 def get_repo_url(artifacthub_config) -> str:
     """Gets the repository url from the ArtifactHub config"""
-    for link in artifacthub_config['links']:
-        if link['name'] == 'Source':
-            return link['url'].replace('https://github.com/', '').partition('/tree')[0]
+    if 'links' in artifacthub_config:
+        for link in artifacthub_config['links']:
+            if link['name'] == 'Source':
+                return link['url'].replace('https://github.com/', '').partition('/tree')[0]
+    else:
+        return None
```

Comment:
It hink you can remove the last 
```
else: return None
```

Python will `return None` by default (also in the case of `links` is available, but no link with name `Source` is available).